src/clr-angular/forms/checkbox/checkbox.ts

```diff
@@ -19,8 +26,25 @@ export class ClrCheckbox extends WrappedFormControl<ClrCheckboxWrapper> {
     @Optional()
     control: NgControl,
     renderer: Renderer2,
-    el: ElementRef
+    el: ElementRef,
+    @Attribute('clrToggle') private toggle: string
```

Comment:
Why this is not `@Input('clrToggle')`? Why treating `clrToggle` as an attribute?

Reply:
I only wanted it to bind once, not be an actual input that could toggle.